antenna/ext/crashstorage_base.py

```diff
@@ -17,14 +17,14 @@ class CrashStorageBase(RequiredConfigMixin):
     def __init__(self, config):
         self.config = config.with_options(self)
 
-    def save_raw_crash(self, raw_crash, dumps, crash_id):
+    def save_raw_crash(self, crash_id, raw_crash, dumps):
```

Comment:
:+1: 

This is a change I proposed in https://github.com/adngdb/crashmill. I also proposed that we separate ``save_raw_crash`` and ``save_dumps``, since the two actions are distinct anyway, and it made the API a little more coherent in my opinion. Cf. https://github.com/adngdb/crashmill/blob/master/crashmill/crash_store.py